pkg/gcrane/gc.go

```diff
@@ -17,62 +17,182 @@ package gcrane
 import (
 	"fmt"
 	"log"
+	"net/http"
+	"time"
 
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/name"
 	"github.com/google/go-containerregistry/pkg/v1/google"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/spf13/cobra"
 )
 
 func init() { Root.AddCommand(NewCmdGc()) }
 
 func NewCmdGc() *cobra.Command {
+	before := ""
+	after := ""
+
 	recursive := false
+	remove := false
+	untag := false
+
 	cmd := &cobra.Command{
 		Use:   "gc",
-		Short: "List images that are not tagged",
+		Short: "Garbage collect images",
 		Args:  cobra.ExactArgs(1),
 		Run: func(_ *cobra.Command, args []string) {
-			gc(args[0], recursive)
+			opts, err := parseOpts(before, after, recursive, remove, untag)
+			if err != nil {
+				log.Fatalln(err)
+			}
+
+			if err := GarbageCollect(args[0], opts); err != nil {
+				log.Fatalln(err)
+			}
 		},
 	}
 
+	// Filters
+	cmd.Flags().StringVarP(&before, "before", "b", "", "Match images uploaded before this time (RFC3339)")
+	cmd.Flags().StringVarP(&after, "after", "a", "", "Match images uploaded after this time (RFC3339)")
```

Comment:
I'm having trouble thinking of a use case for `--after`, maybe just drop this for now until we find one?

Reply:
The scenario I had in mind is something like:

A bug in my Dockerfile means my automation system is producing bad images. While putting out that fire, we fix the bug in the Dockerfile but don't clean up the images immediately. We *know* that there are a few days of garbage images between X and Y. We don't want to delete any of the older good images (so we can rollback) and we don't want to delete any of the new images (because they're in pod).